### ISSUE TITLE
Align Area schema with other related APIs

### ISSUE THREAD

**Issue by bigludo7** [camaraproject/PopulationDensityData#16]
**Problem description**
Probably we should try to align API schema to provide consistency for the developer that will use the CAMARA API. My proposal is to slightly change area structure to get aligned with device location & region user count api.
It will not change the API function.

As of now we have:
```
    Area:
      type: object
      description: >-
        Geographical representation of a polygonal area defined
        by a collection of coordinates forming a closed loop.
      required:
        - boundary
      properties:
        boundary:
          type: array
          description: >-
            Collection of points defining the outer perimeter of the
            area, forming a sequence of vertices connected by straight-line
            segments.
          items:
            $ref: '#/components/schemas/Point'
          minItems: 3
          maxItems: 15
    Point:
      type: object
      description: >-
        Geographical coordinates (latitude, longitude) defining a location in a
        map.
      required:
        - latitude
        - longitude
      properties:
        latitude:
          $ref: '#/components/schemas/Latitude'
        longitude:
          $ref: '#/components/schemas/Longitude'
      example:
        latitude: 39.699938
        longitude: -2.777067
```

to be aligned with other API it could be: 

```
    Area:
      type: object
      properties:
        areaType:
          $ref: "#/components/schemas/AreaType"
      required:
        - areaType
      discriminator:
        propertyName: areaType
        mapping:
          POLYGON: "#/components/schemas/Polygon"

    AreaType:
      type: string
      description: |
        Type of this area.
        POLYGON - The area is defined as a polygon.
      enum:
        - POLYGON

    Polygon:
      allOf:
        - $ref: "#/components/schemas/Area"
        - type: object
          required:
            - boundary
          properties:
            boundary:
              $ref: "#/components/schemas/PointList"

    PointList:
      type: array
      items:
        $ref: "#/components/schemas/Point"
      minItems: 3
      maxItems: 15

    Point:
      type: object
      description: Coordinates (latitude, longitude) defining a location in a map
      required:
        - latitude
        - longitude
      properties:
        latitude:
          $ref: "#/components/schemas/Latitude"
        longitude:
          $ref: "#/components/schemas/Longitude"
      example:
        latitude: 50.735851
        longitude: 7.10066
```

We have adopted this model because it can be easily updated if we want to introduce new area type like circle for example

**Possible evolution**
Update yaml 

**Alternative solution**

**Additional context**

**Comment by sachinvodafone** [comment]
I don't find any issue to restructure it

**Comment by jgarciahospital** [comment]
Agreed and solved in #21

**Comment by bigludo7** [comment]
Checked #21 - Works for me; Thanks
